### DISCUSSION TITLE
O Portugol pode ser usado profissionalmente?

### DISCUSSION THREAD

**Discussion by aledepaulaaa** [dgadelha/Portugol-Webstudio#186] [Dúvidas / Perguntas] [answered]
Olá, gostaria de saber se o Portugol pode ser usado , por exemplo para construir aplicações profissionais.
Lembro que já estudei um pouco a linguagem, no meu inicio mas não tive mais contato com a ferramenta.
Obrigado!

**Comment by dgadelha** [comment, marked as answer]
Não recomendo, é uma linguagem bem limitada

**Reply by aledepaulaaa** [reply]
Entendido @dgadelha 
obrigado pela dica!
